lemmatized_text/test_data.py

```diff
@@ -0,0 +1,130 @@
1913, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[68837]&quot;, &quot;word_normalized&quot;: &quot;cano&quot;}">canō</span><span follower="true">, </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 50164, &quot;resolved&quot;: &quot;resolved-automatic&quot;, &quot;gloss_ids&quot;: &quot;[82748]&quot;, &quot;word_normalized&quot;: &quot;Troiae&quot;}">Trōiae</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 41276, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[79774]&quot;, &quot;word_normalized&quot;: &quot;qui&quot;}">quī</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 1280, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[85717, 79130]&quot;, &quot;word_normalized&quot;: &quot;primus&quot;}">prīmus</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 1239, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[83603, 83604, 66883]&quot;, &quot;word_normalized&quot;: &quot;ab&quot;}">ab</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 1415, &quot;resolved&quot;: &quot;resolved-automatic&quot;, &quot;gloss_ids&quot;: &quot;[85502, 77462]&quot;, &quot;word_normalized&quot;: &quot;oris&quot;}">ōrīs</span><br /><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 27293, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[85202]&quot;, &quot;word_normalized&quot;: &quot;Italiam&quot;}">Ītaliam</span><span follower="true">, </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 1585, &quot;resolved&quot;: &quot;resolved-automatic&quot;, &quot;gloss_ids&quot;: &quot;[72637, 85017]&quot;, &quot;word_normalized&quot;: &quot;fato&quot;}">fātō</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 39952, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[79267]&quot;, &quot;word_normalized&quot;: &quot;profugus&quot;}">profugus</span><span follower="true">, </span><span data-token="{&quot;glossed&quot;: &quot;na&quot;, &quot;lemma_id&quot;: null, &quot;resolved&quot;: &quot;no-lemma&quot;, &quot;gloss_ids&quot;: &quot;[]&quot;, &quot;word_normalized&quot;: &quot;Laviniaque&quot;}">Lāvīniaque</span><span follower="true"> </span><span data-token="{&quot;glossed&quot;: &quot;glossed-automatic&quot;, &quot;lemma_id&quot;: 1299, &quot;resolved&quot;: &quot;no-ambiguity&quot;, &quot;gloss_ids&quot;: &quot;[83198]&quot;, &quot;word_normalized&quot;: &quot;venit&quot;}">vēnit</span>'
```

Comment:
use `transform_data_to_html function`, make sure we have test for `transform_data_to_html` as well